view/activity_activity_form.xml

```diff
@@ -32,8 +32,10 @@ contains the full copyright notices and license terms. -->
     <field name="party"/>
     <label name="resource"/>
     <field name="resource"/>
+    <label name="origin"/>
```

Comment:
As the user will hardly ever fill in origin field, I would put it after location.